eng/pipelines/common/templates/run-test-job.yml

```diff
@@ -190,6 +191,22 @@ jobs:
         displayName: 'product build'
 
 
+    - ${{ if eq(parameters.runtimeFlavor, 'mono') }}:
+      # We need to explictly download CoreCLR for Mono
```

Comment:
I think we want to avoid that and use runtime pack instead (cc @steveisok)

Reply:
This is ok to do temporarily, but will be replaced when the mono runtime pack + sdk modifications are done.